pkg/actors/config.go

```diff
@@ -55,52 +55,62 @@ const (
 	defaultReentrancyStackLimit = 32
 )
 
+// NewConfigOpts contains options for NewConfig.
+type NewConfigOpts struct {
```

Comment:
Should this just be ConfigOpts or if stuttering is ok , ActorConfigOpts to qualify it s bit more? Similarly for other types ... 

What benefit does New provide ?

Methods are named such since they create a new instance to work with ... @ItalyPaleAle ?

Reply:
Done